dev-packages/application-package/src/application-props.ts

```diff
@@ -142,6 +149,7 @@ export type BackendApplicationConfig = RequiredRecursive<BackendApplicationConfi
 export namespace BackendApplicationConfig {
     export const DEFAULT: BackendApplicationConfig = {
         singleInstance: false,
+        frontendConnectionTimeout: 1000
```

Comment:
The timeout starts running when the back end detects that the socket has been closed. So unless we explicitly close the socket (closing the window, for example), this would effectively be 31 seconds. I thought about changing the values for "keepalive" in the socket, but decided to leave things as they are as much as possible.

Reply:
I see, then why not just have it at `0`? `1000` just seems to me like an arbitrarily chosen magic number that doesn't really influence the behavior. There is virtually no difference between 30 and 31 seconds of timeout. I would either go "all in" and wait for an additional 30 seconds, or just don't wait at all beyond the 30 second websocket timeout - not even a second.